server/datastore/mysql/migrations/tables/20250211141712_AddHostCertificatesTable.go

```diff
@@ -0,0 +1,44 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250211141712, Down_20250211141712)
+}
+
+func Up_20250211141712(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE host_certificates (
+	cert_id                     BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
That naming makes it sound like a foreign key, I think I'd go with just `id` for that auto-increment PK? It's also how it's named in the JSON API.